projects/core/src/lib/services/users.data.service.ts

```diff
@@ -41,4 +41,14 @@ export class UsersDataService {
 			})
 		);
 	}
+
+	public saveUserById(user: User): Observable<Subscription | null> {
```

Comment:
is better return Observable User, not Subscription, because there may be inconvenience in using this method

Reply:
switched to Observable<boolean>